app/assets/javascripts/admin/routes/admin-customize-email-style.js.es6

```diff
@@ -0,0 +1,14 @@
+import EmailStyle from "admin/models/email-style";
+import { ajax } from "discourse/lib/ajax";
+
+export default Ember.Route.extend({
+  model() {
+    return ajax("/admin/customize/email_style.json").then(json => {
```

Comment:
Similar to my comment above you should be able to say `this.store.find('emailStyle')` to do this automatically.

Reply:
I tried this, but RestModel doesn't know about singular resources, so I shouldn't make this a RestModel.

Reply:
Ah right, everything needs an ID.